targets/targets.json

```diff
@@ -761,14 +761,18 @@
         "supported_form_factors": ["ARDUINO"],
         "core": "Cortex-M7FD",
         "supported_toolchains": ["ARM", "GCC_ARM", "IAR"],
-        "extra_labels": ["NXP", "MCUXpresso_MCUS", "EVK", "MIMXRT1050", "IMX"],
+        "extra_labels": ["NXP", "MCUXpresso_MCUS", "EVK", "MIMXRT1050", "IMX", "NXP_EMAC"],
```

Comment:
NXP-specific EMAC?

Reply:
This is the location for the NXP EMAC drivers